src/excalidraw-utils/tsconfig.prod.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
this won't be needed since it's. a pure js file right?

Also, we don't need webpack config as well. We can just export `src/index.js` directly and if we want to support older browsers we can just transpile with babel alone.

Reply:
I might be wrong but `src/index.js` file imports TypeScript files (namely `src/scene/export.ts` and `src/appState.ts`) so I guess we still need Webpack and the TypeScript config no?

Reply:
Yup, that can be achieved with `@babel/preset-typescript` as well. 
Since its single pure js functions so we might not need a bundler for code splitting etc.
@dwelle what do you think?
If we want to use webpack we can reuse the same file of excal-embed.

Reply:
Don't have an opinion. If we can remove it, great. Not sure if we can. Your call @ad1992